test/e2e/node/security_context.go

```diff
@@ -146,6 +159,88 @@ var _ = SIGDescribe("Security Context", func() {
 		testPodSELinuxLabeling(f, false, true)
 	})
 
+	setupLocalhostSeccompProfile := func(profiles []seccompProfile) {
+		name := "seccomp-setup-" + string(uuid.NewUUID())
+		hostPathType := v1.HostPathDirectoryOrCreate
+
+		const seccompProfileRoot = "/var/lib/kubelet/seccomp"
```

Comment:
is this path always the same?

Reply:
It can be configured on the node level (deprecated with v1.19.0) and defaults to the kubelet `--root`/seccomp. :)